package.json

```diff
@@ -9,6 +9,7 @@
     "mongoose": "^7.2.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^5.3.0",
```

Comment:
Maybe get rid of "^" symbol in react router dependency so it wont install an updated version